single_tech_samples/fabric/feature_engineering_on_fabric/README.md

```diff
@@ -0,0 +1,419 @@
+  - [Data landing activity](#data-landing-activity)
+  - [Notebook activities](#notebook-activities)
+- [Execute and verify](#execute-and-verify)
+  - [Trigger the data pipeline](#trigger-the-data-pipeline)
+  - [Verify the data lineage in Purview](#verify-the-data-lineage-in-purview)
+  - [Verify the features in Feature Store](#verify-the-features-in-feature-store)
+- [Model training and inferencing](#model-training-and-inferencing)
+  - [Model training](#model-training)
+  - [Model inferencing](#model-inferencing)
+  - [Verify the model training lineage in Purview](#verify-the-model-training-lineage-in-purview)
+- [References](#references)
+
+## Architecture
+
+Here is the high-level architecture diagram:
+
+![Architecture](./images/fs_architecture.png)
+
+This architecture utilizes Microsoft Fabric as the data analytics platform. A [data pipeline](https://learn.microsoft.com/training/modules/use-data-factory-pipelines-fabric/) lands, ingests, and transform the incoming data. The transformed data is registered as [features](https://learn.microsoft.com/azure/machine-learning/concept-what-is-managed-feature-store?view=azureml-api-2#what-are-features) in Azure ML managed feature store. These features are used for model training and inferencing. Concurrently, the data lineage of both the data pipelines and the features is tracked and monitored using Microsoft Purview.
+
+The sample follows a medallion architecture with `landing`, `staging` and `standard` zones created in the __File Section__ of a [lakehouse](https://learn.microsoft.com/fabric/data-engineering/lakehouse-overview) in [Fabric OneLake](https://learn.microsoft.com/fabric/onelake/onelake-overview). For data landing, 'ForEach' activity is used to download multiple files from a public blob storage. The rest of the processing (ingestion, transformation, feature registration, model training, and model inferencing) is done using Fabric 'data notebooks'.
+
+In addition to the main flow, there are optional steps for performing 'exploratory data analysis' and 'data validations' (illustrated by dotted lines in the diagram). These features are currently not covered as part of the step-by-step guide, but the notebooks are available in the repo for reference.
+
+## Environment setup
+
+### Prerequisites
+
+- Access to a [Microsoft Fabric enabled tenant](https://learn.microsoft.com/fabric/admin/fabric-switch#enable-for-your-tenant). You can sign up for Fabric (Preview) trials for a period of 60 days by following [this link](https://learn.microsoft.com/fabric/get-started/fabric-trial).
+- Access to an Azure subscription. You can sign up for a free trial by following [this link](https://azure.microsoft.com/free/).
+- Permissions to [create a service principal](https://learn.microsoft.com/entra/identity-platform/howto-create-service-principal-portal#register-an-application-with-microsoft-entra-id-and-create-a-service-principal) in Microsoft Entra ID.
+
+### Microsoft/Azure resources
```

Comment:
The "/" seems a bit odd. Better just "Microsoft and Azure resources". Or simply Required Resources?

Reply:
Changed to "Required resources"